nightly/complete_nayduck_set.txt

```diff
@@ -0,0 +1,304 @@
+pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions node_restart wipe_data
+pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions node_restart wipe_data --features nightly_protocol --features nightly_protocol_features
+pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions node_restart packets_drop wipe_data
+pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions node_restart packets_drop wipe_data --features nightly_protocol --features nightly_protocol_features
+# pytest --timeout=2000 stress/stress.py 3 2 4 0 staking transactions node_set
+# pytest --timeout=2000 stress/stress.py 3 2 4 0 staking transactions node_set --features nightly_protocol --features nightly_protocol_features
+
+# pytest stress/network_stress.py
+
+# python spec tests
+pytest spec/network/handshake.py
+pytest spec/network/handshake.py --features nightly_protocol --features nightly_protocol_features
+pytest spec/network/peers_request.py
+pytest spec/network/peers_request.py --features nightly_protocol --features nightly_protocol_features
+
+# python adversarial tests
+pytest --timeout=600 adversarial/fork_sync.py
+pytest --timeout=600 adversarial/fork_sync.py --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/wrong_sync_info.py
+pytest adversarial/wrong_sync_info.py --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/malicious_chain.py
+pytest adversarial/malicious_chain.py --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/malicious_chain.py valid_blocks_only
+pytest adversarial/malicious_chain.py valid_blocks_only --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py
+pytest adversarial/start_from_genesis.py --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py overtake
+pytest adversarial/start_from_genesis.py overtake --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py doomslug_off
+pytest adversarial/start_from_genesis.py doomslug_off --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py overtake doomslug_off
+pytest adversarial/start_from_genesis.py overtake doomslug_off --features nightly_protocol --features nightly_protocol_features
+
+# python test for enable inflation
+pytest sanity/enable_inflation.py
+pytest sanity/enable_inflation.py --features nightly_protocol --features nightly_protocol_features
+
+# python upgradable test
+# upgradable.py moves `near` binary, and must be the last python test in the set
+pytest --timeout=600 sanity/upgradable.py
+pytest --timeout=600 sanity/upgradable.py --features nightly_protocol --features nightly_protocol_features
```

Comment:
nightly isn't always backward compatible so we should not enable nightly features for this test

Reply:
Done